src/Cake.Npm/AddUser/NpmAddUserSettings.cs

```diff
@@ -0,0 +1,96 @@
+﻿namespace Cake.Npm.AddUser
+{
+    using System;
+    using System.Linq;
+    using Core;
+    using Core.IO;
+
+    /// <summary>
+    /// Contains settings used by <see cref="NpmAddUser"/>.
+    /// </summary>
+    public class NpmAddUserSettings : NpmSettings
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="NpmAddUserSettings"/> class.
+        /// </summary>
+        public NpmAddUserSettings()
+            : base("adduser")
+        {
+        }
+
+        /// <summary>
+        /// Gets or sets a value indicating which registry we should point to. Defaulted to whatever the NPM configuration is.
+        /// </summary>
+        /// <value>
```

Comment:
I would also be happy with only the `summary` part, as long as default value is documented in the summary. I think we never have documented the value in other classes